makefiles/py.inc.mk

```diff
@@ -0,0 +1,19 @@
+# generate .py.h header files of .py files
+PY_HEADER_DIR ?= $(BINDIR)/$(MODULE)py_h
+PY = $(wildcard *.py)
+PY_H := $(PY:%.py=$(PY_HEADER_DIR)/%.py.h)
+
+ifneq (, $(PY_H))
+	CFLAGS += -I$(PY_HEADER_DIR)
+endif
+
+$(PY_HEADER_DIR):
+	@mkdir -p $@
+
+$(PY_H): $(PY_HEADER_DIR)
+$(PY_H): $(PY_HEADER_DIR)/%.py.h: %.py $(PY)
```

Comment:
Why does this depend on $(PY)

Reply:
So it triggers a rebuild when the python files change.

Reply:
But why you want _any_ change in a python file trigger a rebuild in _all_ of them?